scipy/stats/_continuous_distns.py

```diff
@@ -1732,6 +1732,15 @@ def _ppf(self, q, c):
         fac = np.power(-np.log(fac), 1.0 / c)
         return np.where(q > 0.5, fac, -fac)
 
+    def _sf(self, x, c):
+        Cx1 = 0.5 * np.exp(-abs(x)**c)
+        return np.where(x > 0, Cx1, 1 - Cx1)
+
+    def _isf(self, q, c):
+        fac = 2. * np.where(q <= 0.5, q, 1. - q)
+        fac = np.power(-np.log(fac), 1.0 / c)
+        return np.where(q > 0.5, -fac, fac)
+
```

Comment:
```suggestion
    def _sf(self, x, c):
        p = weibull_min._sf(x, c) / 2
        return np.where(x > 0, p, 1 - p)

    def _isf(self, q, c):
        i = (q <= 0.5)
        fac = 2. * np.where(i, q, 1 - q)
        fac = weibull_min._isf(fac, c)
        return np.where(i, fac, -fac)
```

This fails tests until `_sf` is overridden for `weibull_min`. Following the style of its `ppf`,
```python3
    def _isf(self, q, c):
        return pow(-np.log(q), 1.0/c)
```
or 
```python3
    def _isf(self, q, c):
        return (-np.log(q))**(1.0/c)
```
would be fine, too.

Reply:
Let's not accept and silently revert changes in the future, please.